web/src/ee/features/evals/components/evaluator-detail.tsx

```diff
@@ -63,87 +73,109 @@ export const EvaluatorDetail = () => {
 
   return (
     <FullScreenPage>
-      <>
-        <Header
-          title={evaluator.data?.id ?? "Loading..."}
-          breadcrumb={[
-            {
-              name: "Evaluators",
-              href: `/project/${router.query.projectId as string}/evals`,
-            },
-            { name: evaluator.data?.id },
-          ]}
-          actionButtons={
-            <>
-              <DeactivateEvaluator
-                projectId={projectId}
-                evaluator={evaluator.data ?? undefined}
-                isLoading={evaluator.isLoading}
+      <Header
+        title={evaluator.data ? `Evaluator ${evaluator.data.id}` : "Loading..."}
+        breadcrumb={[
+          {
+            name: "Evaluators",
+            href: `/project/${router.query.projectId as string}/evals`,
+          },
+          { name: evaluator.data?.id },
+        ]}
+        actionButtons={
+          <>
+            <StatusBadge
+              type={evaluator.data?.status.toLowerCase()}
+              isLive
+              className="max-h-8"
+            />
+            <DeactivateEvaluator
+              projectId={projectId}
+              evaluator={evaluator.data ?? undefined}
+              isLoading={evaluator.isLoading}
+            />
+            {evaluator.data && (
+              <DetailPageNav
+                key="nav"
+                currentId={encodeURIComponent(evaluator.data.id)}
+                path={(entry) =>
+                  `/project/${projectId}/evals/${encodeURIComponent(entry.id)}`
+                }
+                listKey="evals"
               />
-              {evaluator.data && (
-                <DetailPageNav
-                  key="nav"
-                  currentId={encodeURIComponent(evaluator.data.id)}
-                  path={(entry) =>
-                    `/project/${projectId}/evals/${encodeURIComponent(entry.id)}`
-                  }
-                  listKey="evals"
+            )}
+          </>
+        }
+      />
+      {existingEvaluator && (
+        <TableWithMetadataWrapper
+          tableComponent={
+            <EvalLogTable
+              projectId={projectId}
+              jobConfigurationId={existingEvaluator.id}
+            />
+          }
+          cardTitleChildren={
+            <div className="flex w-full flex-row items-center justify-between">
+              <span>Evaluator configuration</span>
+              <Dialog open={isEditOpen} onOpenChange={setIsEditOpen}>
+                <DialogTrigger asChild>
+                  <Button variant="outline" className="flex items-center gap-2">
+                    <Edit className="h-5 w-5" />
```

Comment:
Nit: this should be `h-4 w-4`, we should really just set a helper for that though, will add to my backlog

Reply:
Done, thanks 👍🏾